src/server/protocol.ts

```diff
@@ -597,6 +599,28 @@ export interface GetApplicableRefactorsResponse extends Response {
     body?: ApplicableRefactorInfo[];
 }
 
+/**
+ * Request refactorings at a given position or selection area to move to an existing file.
+ */
+export interface GetMoveToRefactoringFileSuggestionsRequest extends Request {
+    command: CommandTypes.GetMoveToRefactoringFileSuggestions;
+    arguments: GetMoveToRefactoringFileSuggestionsRequestArgs;
+}
+export type GetMoveToRefactoringFileSuggestionsRequestArgs = FileLocationOrRangeRequestArgs & {
+    triggerReason?: RefactorTriggerReason;

```

Comment:
Can we drop this property or would we return different results depending on how the refactoring is triggered?

Reply:
Yes, it's safe to delete it.